solr/modules/llm/build.gradle

```
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Indexing/Query time integration with LLM'
+
+dependencies {
+  implementation project(':solr:core')
+  implementation project(':solr:solrj')
+
+  implementation 'org.apache.lucene:lucene-core'
+  
+  implementation 'dev.langchain4j:langchain4j-hugging-face:0.35.0'
+  implementation 'dev.langchain4j:langchain4j-mistral-ai:0.35.0'
+  implementation 'dev.langchain4j:langchain4j-open-ai:0.35.0'
+  implementation 'dev.langchain4j:langchain4j-cohere:0.35.0'
+  implementation 'dev.langchain4j:langchain4j:0.35.0'
```

Wondering if it's possible to omit the versions here, in anticipation of future upgrades etc?
```suggestion
  implementation 'dev.langchain4j:langchain4j-hugging-face'
  implementation 'dev.langchain4j:langchain4j-mistral-ai'
  implementation 'dev.langchain4j:langchain4j-open-ai'
  implementation 'dev.langchain4j:langchain4j-cohere'
  implementation 'dev.langchain4j:langchain4j'
```